code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -169,119 +187,66 @@
-	var/list/adjacent_turfs = atmos_adjacent_turfs
-	var/datum/excited_group/our_excited_group = excited_group
-	var/adjacent_turfs_length = LAZYLEN(adjacent_turfs)
-	var/cached_atmos_cooldown = atmos_cooldown + 1
-
-	var/planet_atmos = planetary_atmos
-	if (planet_atmos)
-		adjacent_turfs_length++
-
-	var/datum/gas_mixture/our_air = air
-
-	#ifdef TRACK_MAX_SHARE
-	max_share = 0 //Gotta reset our tracker
-	#endif
-
-	for(var/t in adjacent_turfs)
-		var/turf/open/enemy_tile = t
-
-		if(fire_count <= enemy_tile.current_cycle)
+/turf/open/proc/equalize_pressure_in_zone(cyclenum)
+/*
+/turf/open/proc/consider_firelocks(turf/T2)
+	var/reconsider_adj = FALSE
+	for(var/obj/machinery/door/firedoor/FD in T2)
+		if((FD.flags_1 & ON_BORDER_1) && get_dir(T2, src) != FD.dir)
+			continue
+		FD.emergency_pressure_stop()
+		reconsider_adj = TRUE
+	for(var/obj/machinery/door/firedoor/FD in src)
+		if((FD.flags_1 & ON_BORDER_1) && get_dir(src, T2) != FD.dir)
 			continue
-		enemy_tile.archive()
-
-	/******************* GROUP HANDLING START *****************************************************************/
-
-		var/should_share_air = FALSE
-		var/datum/gas_mixture/enemy_air = enemy_tile.air
-
-		//cache for sanic speed
-		var/datum/excited_group/enemy_excited_group = enemy_tile.excited_group
-
-		if(our_excited_group && enemy_excited_group)
-			if(our_excited_group != enemy_excited_group)
-				//combine groups (this also handles updating the excited_group var of all involved turfs)
-				our_excited_group.merge_groups(enemy_excited_group)
-				our_excited_group = excited_group //update our cache
-			should_share_air = TRUE
-
-		else if(our_air.compare(enemy_air))
-			if(!enemy_tile.excited)
-				SSair.add_to_active(enemy_tile)
-			var/datum/excited_group/EG = our_excited_group || enemy_excited_group || new
-			if(!our_excited_group)
-				EG.add_turf(src)
-			if(!enemy_excited_group)
-				EG.add_turf(enemy_tile)
-			our_excited_group = excited_group
-			should_share_air = TRUE
-
-		//air sharing
-		if(should_share_air)
-			var/difference = our_air.share(enemy_air, adjacent_turfs_length)
-			if(difference)
-				if(difference > 0)
-					consider_pressure_difference(enemy_tile, difference)
-				else
-					enemy_tile.consider_pressure_difference(src, -difference)
-			LAST_SHARE_CHECK
-
-
-	/******************* GROUP HANDLING FINISH *********************************************************************/
-
-	if (planet_atmos) //share our air with the "atmosphere" "above" the turf
-		var/datum/gas_mixture/G = new
-		G.copy_from_turf(src)
-		G.archive()
-		if(our_air.compare(G))
-			if(!our_excited_group)
-				var/datum/excited_group/EG = new
-				EG.add_turf(src)
-				our_excited_group = excited_group
-			our_air.share(G, adjacent_turfs_length)
-			LAST_SHARE_CHECK
-
-	our_air.react(src)
+		FD.emergency_pressure_stop()
+		reconsider_adj = TRUE
+	if(reconsider_adj)
+		T2.ImmediateCalculateAdjacentTurfs() // We want those firelocks closed yesterday.
+		*/
```

Comment:
Why is this code commented here?

Reply:
Wait.  Isn't this the whack_a_mole_part stuff?  I am still downloading this thing